src/compiler.js

```diff
@@ -0,0 +1,36 @@
+import { resolve } from 'path';
+import promisify from 'es6-promisify';
+import rimraf from 'rimraf';
+import prerender from './lib/prerender';
+import webpackConfig from './lib/webpack-config';
+import runWebpack, { showStats } from './lib/run-webpack';
+
+class Compiler {
```

Comment:
Curious about everyone's opinions on this, but I wondered if this could have been done more directly via a closure + exported object (just a POJO factory).  If everyone is good with classes I don't have a strong opinion here, just it's the first class into this codebase that isn't specifically for Webpack, so I thought I'd ask :)